src/value.rs

```diff
@@ -101,6 +104,10 @@ pub enum Value {
     #[cfg_attr(docsrs, doc(cfg(feature = "with-uuid")))]
     Uuid(Option<Box<Uuid>>),
 
+    #[cfg(feature = "with-uuid-1")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "with-uuid-1")))]
+    Uuid1(Option<Box<Uuid1>>),
```

Comment:
If the behaviour and API between `uuid v8` and `uuid v1` remains the same. We should switch the version of `uuid` to v1 if `with-uuid-1` feature is enabled, instead of introducing a new UUID variant in `Value` enum.

Reply:
I mean swapping the version of `uuid`.
- If `with-uuid` is enabled but not `with-uuid-1`, then we import `use uuid::Uuid;`
- If `with-uuid-1` is enabled, then we import `use uuid_1::Uuid;`

i.e. the same old `Uuid(Option<Box<Uuid>>)` unchanged. But we swap the inner `uuid` to either v0.8 / v1.0